geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -104,6 +109,8 @@
     helpMap.put("bind-address",
```

Comment:
I'd love to break these Launcher classes up into smaller classes. I'd also like to move them from org.apache.geode.distributed -- I don't think that pkg really makes sense for these classes to live in.